async_producer_test.go

```diff
@@ -1248,21 +1248,21 @@ func ExampleAsyncProducer_select() {
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, os.Interrupt)
 
-	var enqueued, errors int
+	var enqueued, producerError int
 ProducerLoop:
 	for {
 		select {
 		case producer.Input() <- &ProducerMessage{Topic: "my_topic", Key: nil, Value: StringEncoder("testing 123")}:
 			enqueued++
 		case err := <-producer.Errors():
 			log.Println("Failed to produce message", err)
-			errors++
+			producerError++
 		case <-signals:
 			break ProducerLoop
 		}
 	}
 
-	log.Printf("Enqueued: %d; errors: %d\n", enqueued, errors)
+	log.Printf("Enqueued: %d; producerError: %d\n", enqueued, producerError)
```

Comment:
It doesn't seem necessary to replace errors here, 
this reads better:
```go
log.Printf("Enqueued: %d; errors: %d\n", enqueued, producerErrors)
```

Reply:
That's IDE refactoring, will fix it